### DISCUSSION TITLE
Does the Built-in Auto Backup Purge itself?

### DISCUSSION THREAD

**Discussion by E-Hain** [inventree/InvenTree#7108] [Q&A]
Does the Built-in Auto Backup Purge itself automatically too? So that there aren't a bunch of backups?

**Comment by matmair** [comment]
It should, default is 10 but that could be configured with a custom settings.py file - see docs of the underlying lib [here](https://django-dbbackup.readthedocs.io/en/master/configuration.html#dbbackup-cleanup-keep-and-dbbackup-cleanup-keep-media)

**Reply by E-Hain** [reply]
Is there a default settings.py to reference?

**Reply by matmair** [reply]
No, our settings is 1.3k lines long, that is not helpful
